notebooks/notebooks/iclr2022/win-rate-vs-visits.ipynb

```diff
@@ -0,0 +1,460 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #38.        minor_ticks = utils.flatten_2d_list(
</pre><p>I found this line difficult to understand but it's actually doings something quite simple (add 9 ticks between each major tick in intervals of 10) -- add comment or edit to be more readily comprehensible?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>